packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -410,7 +410,8 @@ class TextSelectionOverlay {
 
   /// Shows the toolbar by inserting it into the [context]'s overlay.
   void showToolbar() {
-    assert(_toolbar == null);
+    if (_toolbar != null)
+      return;
```

Comment:
Did you change this because you were getting lots of assertions thrown when they shouldn't be?  I was seeing this as well at times while working on selection handles stuff but I thought I fixed it.  Are you sure it's ok to remove the assert?

Reply:
I remember i happened on a very edgy case of textfield which i forgot how to reproduce or it has been fixed in my pr. I will remove this for now